update.php

```diff
@@ -310,7 +310,7 @@ function update_1031() {
 	$r = q("SELECT `id`, `object` FROM `item` WHERE `object` != '' ");
 	if($r && dbm::is_result($r)) {
```

Comment:
Standards: Can you please add a space after the `if` and simplify the condition?